lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,41 @@
-module Minesweeper_2pl
+module Minesweeper
   class App
-    def call
-      "Hello World"
+    BOMB_PERCENT = 0.75
+    MAX_ROW_NUM = 20
+
+    attr_accessor :game, :cli
+
+    def initialize
+      cli = CLI.new
+      self.cli = cli
+      game_config = cli.start
```

Comment:
Variable naming can be a clue to more descriptive method names. If `CLI#start` returns a "game config," could there be a more descriptive name for what "start" is doing?

Reply:
(More descriptive names are often also more boring)